doc/nrf/ug_multi_image.rst

```diff
@@ -152,6 +153,37 @@ See the following example:
 
 A *domain* is well-defined if there is the ``CONFIG_DOMAIN_${DOMAIN}_BOARD`` configuration option in Kconfig.
 
+.. _ug_multi_image_add_child_image_oot:
+
+Adding a child image using Zephyr modules
+=========================================
+
+Any call to ``add_child_image`` must be done *after* :file:`nrf/cmake/extensions.cmake` is invoked, but *before* :file:`multi_image.cmake` is invoked.
+In some scenarios, this is not possible without modifying the NCS build code, for example, from top-level sample files and project :file:`CMakeLists.txt` files.
+
+To avoid this issue, use the *Modules* mechanism provided by the Zephyr build system.
+The following example shows how to add the required module from a top-level sample :file:`CMakeLists.txt`.
+
+.. literalinclude:: ../../samples/nrf5340/multicore/CMakeLists.txt
+    :language: cmake
+    :start-at: cmake_minimum_required
+    :end-at: target_sources
+
+A :file:`zephyr/module.yml` file is needed at the base of the added module.
+The following example specifies only the path to the :file:`CMakeLists.txt` of the new module.
+See :ref:`modules` for more details.
+
+.. literalinclude:: ../../samples/nrf5340/multicore/zephyr/module.yml
+    :language: cmake
+
+The :file:`CMakeLists.txt` located in the directory pointed to by :file:`zephyr/module.yml` will be invoked when ``add_child_image`` can be invoked.
+
+.. literalinclude:: ../../samples/nrf5340/multicore/aci/CMakeLists.txt
+    :language: cmake
+    :start-at: add_child_image
+    :end-after: BOARD ${CONFIG_DOMAIN_CPUNET_BOARD}
```

Comment:
```suggestion
    :end-after: BOARD ${CONFIG_DOMAIN_CPUNET_BOARD}
```

It looks like it is not recognizing this subdirective. From a brief search, there are no other instances of this subdirective being used anywhere else in the docs. Maybe it was not implemented.

Could you switch to :end-at: or :end-before: ? (adjusting the related string accordingly)